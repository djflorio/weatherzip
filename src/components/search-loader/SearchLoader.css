.search-loader {
  padding: 30px 0;
  text-align: center;
}

.search-loader__text {
  margin: 0;
  color: #fff;
  font-weight: 800;
}