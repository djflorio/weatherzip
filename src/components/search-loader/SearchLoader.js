// Third Party
import React from 'react';

// Assets
import './SearchLoader.css';

// Components
import { HourGlass } from '../spinners/Spinners';


const SearchLoader =() => (
  <div className="search-loader">
    <HourGlass />
    <p className="search-loader__text">Getting your forecast</p>
  </div>
);

export default SearchLoader;