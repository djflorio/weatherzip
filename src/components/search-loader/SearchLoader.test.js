import SearchLoader from './SearchLoader';

it("renders correctly", () => {
  const wrapper = shallow(
    <SearchLoader />
  );

  expect(wrapper).toMatchSnapshot();
});