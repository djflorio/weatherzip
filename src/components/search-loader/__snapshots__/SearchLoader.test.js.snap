// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="search-loader"
>
  <HourGlass />
  <p
    className="search-loader__text"
  >
    Getting your forecast
  </p>
</div>
`;
